api/src/main/resources/workbench-api.yaml

```diff
@@ -5674,6 +5674,18 @@ definitions:
             description: >
               Whether any modules have expired. Each expired module will have
               the corresponding hasExpired flag set in the modules array.
+      accessModules:
+        type: object
+        properties:
+          modules:
+            type: array
+            items:
+              "$ref": "#/definitions/AccessModuleStatus"
+          anyModuleHasExpired:
+            type: boolean
+            description: >
+              Whether any modules have expired. Each expired module will have
+              the corresponding hasExpired flag set in the modules array.
```

Comment:
this text is no longer true.  Are you planning to keep the idea of `hasExpired` or remove it?

Reply:
updated

Reply:
Thanks!  Could you update it a little further, something like: "Whether any modules have expired. Expiration information is also included with each module in the array; this field is provided for convenience."